src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -670,6 +775,7 @@ def __call__(
         lora_scale = (
             self.joint_attention_kwargs.get("scale", None) if self.joint_attention_kwargs is not None else None
         )
+        do_true_cfg = true_cfg_scale > 1 or negative_prompt is not None
```

Comment:
```suggestion
        do_true_cfg = true_cfg_scale > 1 and negative_prompt is not None
```

i thought it should be `and` no? at least for `true_cfg_scale > 1` it is not taking an effect